docs/CLOUD_RECORDING.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Build a Cloud Recording Backend with Astro
+description: Learn how to build a backend to initiate cloud recording using Astro.
+---
+
+Not everyone can join every meeting. For example, I struggle to join the boring ones. For people like me having a recording of the meeting is a game changer.
+
+This is part two of a series about building a backend for your Agora video calls. The first guide was about building a [token generator](https://github.com/AgoraIO-Community/astro-backend/blob/main/docs/TOKENS.md). In this guide, we will use that token generator and add cloud recording to our video call.
+
+
+## Prerequisites
+1. NodeJS and Astro installed.
+2. A developer account with [Agora](https://console.agora.io/).
+3. A token generator built with Astro. Find the [guide here](https://github.com/AgoraIO-Community/astro-backend/blob/main/docs/TOKENS.md).
+4. AWS S3 Storage Bucket.
+
+## Project Setup
+This guide will build upon the [token generator guide](https://github.com/AgoraIO-Community/astro-backend/blob/main/docs/TOKENS.md). The token generator guide walks through the process of building the `api/tokens.json` endpoint which returns a token used to secure your video calls. This endpoint uses a `handleGenerateToken` function which handles the token generation logic. We will be using this function directly within the cloud recording endpoint, since it is contained within the same backend server.
+
+From the previous guide, you should have `APP_ID` and the `APP_CERTIFICATE` in your environment variables. We will add more environment variables throughout this guide, but for now we can get started.
+
+
+## Cloud Recording Overview
+Cloud Recording is a RESTful API provided by Agora that allows you to record your video calls into a storage provider of your choice. This recording should be triggered by your backend so that sensitive data isn't being passed to and from the application directly. 
+
+The flow for a successful Cloud Recording session is:
+1. Acquire the an Agora cloud recording resource.
+2. Start the cloud recording using a custom configuration. This configuration includes the necessary keys to allow writing files to your storage provider.
```

Comment:
These two sentences could be a bit more clear. instead of custom config, be more specific and call it a storage configuration, and drop the repetition of the word "configuration"

> Start the cloud recording using a custom storage configuration. This includes the necessary keys to allow writing files to your storage provider.

Reply:
Done